### ISSUE TITLE
Scaling to zero using the faas-idler

### ISSUE THREAD

**Issue by mabuaisha** [hashicorp/faas-nomad#85]
Hi,

What is the timeline of adding `faas-idler` to faas-nomad? if this is going to take so long, is there a workaround solution for to scale to zero when for inactivity ?

**Comment by acornies** [comment]
Hi @mabuaisha, thanks for reaching out:
The faas-idler should work with the faas-nomad provider as is since the provider itself only deals with CRUD operations to the containers.
Looking at: https://github.com/openfaas/faas-netes/blob/master/yaml/faas-idler-dep.yml we could port this over to Nomad HCL and since the ideler only needs access to the gateway and provometheus, the gateway will invoke the nomad-provider's delete handler etc.
In short, no changes are required on the provider to get this working.

**Comment by mabuaisha** [comment]
@acornies Thanks for reply, Glad to hear that, If I understand you correctly then all requried is try to build a Nomad HCL from https://github.com/openfaas/faas-netes/blob/master/yaml/faas-idler-dep.yml  and deploy it to Nomad cluster and it will be fine

Actually I'm new to Nomad and will try to port this it HCL, but if you could guide me here that would be helpful

**Comment by acornies** [comment]
Take a look at: https://github.com/acornies/faas-nomad-demos. I have a few HCLs for the OpenFaaS project and this would be a good addition!

**Comment by mabuaisha** [comment]
@acornies Thanks, sure will take a look there
